recipes/cgal/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "4.8.1" %}
+
+package:
+  name: cgal
+  version: {{ version }}
+
+source:
+    fn: CGAL-{{ version }}.tar.xz
+    url: https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-{{ version }}/CGAL-{{ version }}.tar.xz
+    md5: 1cbcb3f251822766ed3d744f923c782d
+
+build:
+  number: 1
+# needs GMP on windows
```

Comment:
We have `mpir` on Windows if you want to give that a try. It is basically a cross-platform `gmp`. Though we normally use it on Windows only.